.github/workflows/main.yaml

```diff
@@ -336,6 +336,37 @@ jobs:
         uses: ./src/.github/actions/api_leak_detector
         if: inputs.run_api_leak_detector
 
+  package-script-test:
+    needs: [initialize, build]
+    if: always() &&
+        (
+          (inputs.platform == 'android-arm' && github.event.pull_request.draft == false)
```

Comment:
I wouldn't limit it to android. Lets just run it for all platforms, and for all gold + qa configs. If the script doesn't know what to do with Evergreen or linux, we'll fix or stub those things in the script.